### ISSUE TITLE
남은 작업 분담

### ISSUE THREAD

**Issue by Haeuncs** [DDD-4/Spark-iOS#86]
1. 네트워크 실패 시 에러처리 (단순 실패 및 현재 기기의 네트워크 상태에 따른 표시)
2. 로그인 처리 (로그인 처리 및 쿠키 등 처리 작업, 쿠키를 쓴다면 쿠키 만료시 재로그인 로직 등)
3. 탈퇴하기, 회원정보 조회, 수정

backlog
4. 남은 Alert UI 들 개발, 로딩 UI 개발 (? 디자인이 나온다면?)
5. Add User Tracking system (만약 추가한다면?)

@Odyflame 님 생각나시는 다른 남은 작업이 있으면 comment 추가해주세요

**Comment by Odyflame** [comment]
네 작업분담 확인했습니다. 지금 당장은 남은 작업에 대해 떠오르는게 없는데 일단 지금 4개 먼저 하면서 필요한게 있으면 추가하는게 좋을것같습니다
